From patches Mon Sep 17 00:00:00 2001
From: Jan Kiszka <jan.kiszka@web.de>
To: Anthony Liguori <aliguori@us.ibm.com>, Paolo Bonzini <pbonzini@redhat.com>,
 Jan Kiszka <jan.kiszka@web.de>
Cc: Anthony Liguori <aliguori@us.ibm.com>, malc <av1474@comtv.ru>,
 qemu-devel <qemu-devel@nongnu.org>
Date: Wed, 01 May 2013 16:14:34 +0200
Message-id: 5181234A.6060504@web.de
Subject: [PATCH 1/1] audio: Enable all cards

From: Jan Kiszka <jan.kiszka@siemens.com>

...or they will bitrot to death.

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
---
 default-configs/sound.mak |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/default-configs/sound.mak b/default-configs/sound.mak
index ff69c4d..4f22c34 100644
--- a/default-configs/sound.mak
+++ b/default-configs/sound.mak
@@ -1,4 +1,4 @@
 CONFIG_SB16=y
-#CONFIG_ADLIB=y
-#CONFIG_GUS=y
-#CONFIG_CS4231A=y
+CONFIG_ADLIB=y
+CONFIG_GUS=y
+CONFIG_CS4231A=y
-- 
1.7.3.4

